src/Core/System/SystemConfig/SystemConfigService.php

```diff
@@ -79,7 +81,10 @@ public function get(string $key, ?string $salesChannelId = null)
 
         foreach ($parts as $part) {
             if (!\is_array($pointer)) {
-                return null;
+                $event = new SystemConfigLoadedEvent($key, null, $salesChannelId);
```

Comment:
I am not sure, if this should be here, as the result will be cached. Is it okay to cache decrypted values 🤔

Reply:
@shyim Isn't it the config cached in the ConfigLoader service? From what I can see the `get` method of the SystemConfigService isn't cached but I could be wrong?

Reply:
ahh nvm looked into the wrong class 😅

Reply:
but maybe the event name is misleading but I just couldn't think of a better one.